conf/broker.conf

```diff
@@ -207,6 +207,9 @@ tlsTrustCertsFilePath=
 # Accept untrusted TLS certificate from client
 tlsAllowInsecureConnection=false
 
+# Specify whether Client certificates are required for TLS
+# Reject the Connection if the Client Certificate is not trusted.
+tlsReqTrustedClientCertOnConnect=false
```

Comment:
nit: please use full work instead of abbreviation to improve readabilty `tlsRequireTrustedClientCertOnConnect`